docs/admin/env_variables.md

```diff
@@ -46,6 +46,10 @@ Environment variables in this section only apply to the server side.
 |DAOS\_SCHED\_RELAX\_MODE|The mode of CPU relaxing on idle. "disabled":disable relaxing; "net":wait on network request for INTVL; "sleep":sleep for INTVL. STRING. Default to "net"|
 |DAOS\_SCHED\_RELAX\_INTVL|CPU relax interval in milliseconds. INTEGER. Default to 1 ms.|
 |DAOS\_STRICT\_SHUTDOWN|Use the strict mode when shutting down engines. BOOL. Default to 0. In the strict mode, when certain resource leaks are detected, for instance, the engine will raise an assertion failure.|
+|DAOS\_DTX\_AGG\_THD\_CNT|DTX aggregation count threshold. The valid range is [2^20, 2^24]. The default value is 2^19*7.|
+|DAOS\_DTX\_AGG\_THD\_AGE|DTX aggregation age threshold in seconds. The valid range is [210, 1830]. The default value is 630.|
+|DAOS\_DTX\_RPC\_HELPER\_THD|DTX RPC helper threshold. The valid range is [18, unlimited). The default value is 513.|
```

Comment:
I'm wondering if it's a good idea to export all these variables to user. If they are for developers only, they shouldn't be put here.

Reply:
Different from "reclaim property", the aggregation DTX committed entries may affect the correctness of handling resent RPC. For example, if the admin can allow (or guarantee) at most 1 minute resent internal, then he/she can set the DTX aggregation age threshold as a bit larger than 60 seconds, such as 63. That is more flexible for the admin to control the system based on their real environment. Anyway, if we want to implement the TOOP (from Liang's design), these environment variables may be useless.